tools-local/osgroups.props

```diff
@@ -28,12 +28,14 @@
       <Imports>Unix</Imports>
       <TargetsUnix>true</TargetsUnix>
       <TargetsFreeBSD>true</TargetsFreeBSD>
+      <PackageTargetRuntime>freebsd</PackageTargetRuntime>
     </OSGroups>
     <OSGroups Include="NetBSD">
       <Imports>Unix</Imports>
       <TargetsUnix>true</TargetsUnix>
       <TargetsNetBSD>true</TargetsNetBSD>
+      <PackageTargetRuntime>netbsd</PackageTargetRuntime>
```

Comment:
Are you trying to support netbsd as well? I don't see it added to the runtimegroups.props

Reply:
not at this moment. I thought it would be one less place to touch in future and the NetBSD entry is already there. We currently default to 'win' and that is clearly wrong.